models/repo.go

```diff
@@ -1924,7 +1924,7 @@ func GitFsck() {
 			repo := bean.(*Repository)
 			repoPath := repo.RepoPath()
 			if err := git.Fsck(repoPath, setting.Cron.RepoHealthCheck.Timeout, setting.Cron.RepoHealthCheck.Args...); err != nil {
-				desc := fmt.Sprintf("Fail to health check repository '%s': %v", repoPath, err)
+				desc := fmt.Sprintf("Health check failed on repository '%s': %v", repoPath, err)
```

Comment:
I prefer starts with `Failed to xxx`, how about `Failed to perform health check on xxx`?

Reply:
Sure, that sounds good too!